roles/idm/README.md

```diff
@@ -0,0 +1,12 @@
+#CREATE IDENTITIES IN AN EXISTING IDM 
+
+An ansible role that will consume a google sheet and insert that data into and existing IDM (IPA). the format of the sheet should include a header row
+
+```
+user_name, first_name, last_name, email, expiration_date, group
```

Comment:
What happens if we add additional columns that shouldn't be processed? For example; let's say we'd want to add an "Additional Info" column that we can use to just track info within the sheet. Will the implementation just ignore that column, or does it cause problems? I can just try it, but haven't had a chance to do so yet, so wanted to post my question while looking over the code.